home-docs/starting-the-project.md

```diff
@@ -0,0 +1,22 @@
+## What you need to know about the project
+
+### Starting the project locally
+When you have successfully cloned the project, you need to do some steps to start the application.
+Make sure that you have Java 11 installed.
+Firstly, you need create database `home` using PostgreSQL. 
+Then you need to create environmental variables that are defined in `application.properties` file.  
+If you want to fill database with data, you need to activate `dev` profile 
+and use `spring.jpa.hibernate.ddl-auto=create` in `application.properties`.
```

Comment:
I don't think that `spring.jpa.hibernate.ddl-auto=create` should be required here. Only migrations have to be a source of our DDL even for local development.

Reply:
Yes, it makes sense.